tests/test_app.py

```diff
@@ -186,6 +192,18 @@ def test_postcode_bad_space_search_success(self):
         postcode = sanitise_postcode(search_term)
         assert postcode == 'PL9 8TB'
 
+    @mock.patch('requests.get', return_value=fake_postcode_search)
+    def test_postcode_search_with_page_calls_api_correctly(self, mock_get):
+        self.app.get('/title-search/PL9%207FN?page=23')
+        mock_get.assert_called_with('http://landregistry.local:8004/title_search_postcode/PL9 7FN',
+                                    params={'page': 23})
+
+    @mock.patch('requests.get', return_value=fake_address_search)
```

Comment:
These tests are good. I've also been thinking about these cases: page being less then 1, not a number, greater than the number of pages (?). Do we have tests that check if we present what the API gave us?